Packs/GoogleDocs/Integrations/GoogleDocs/README.md

```diff
@@ -0,0 +1,158 @@
+Use the Google Docs integration to create and modify Google Docs documents.
+This integration was integrated and tested with version xx of GoogleDocs
```

Comment:
Please update the version of GoogleDocs.
If not applicable - remove this line.

Reply:
Done, deleted the line because I could not find the relevant GoogleDocs's version.